cognite_toolkit/_cdf_tk/utils/aggregators.py

```diff
@@ -60,8 +123,26 @@ def __init__(self, client: ToolkitClient) -> None:
     def display_name(self) -> str:
         return "Assets"
 
-    def count(self) -> int:
-        return self.client.assets.aggregate_count()
+    def count(self, hierarchy: str | None = None, data_set_external_id: str | None = None) -> int:
+        return self.client.assets.aggregate_count(filter=self._create_hierarchy_filter(hierarchy, data_set_external_id))
+
+    @classmethod
+    def _create_hierarchy_filter(
```

Comment:
These `_create_hierarchy_filter` functions should be unit tested with the nine options (None, "", "non-empty") x s (None, "", "non-empty")